cmd/argocd/commands/admin/settings.go

```diff
@@ -406,6 +406,7 @@ argocd admin settings resource-overrides ignore-differences ./deploy.yaml --argo
 					return
 				}
 
+				// TODO validate if this IgnoreNormalizer needs to ignore managed fields as well.
```

Comment:
I think there is no need to ignore managed fields here. The `argocd admin settings resource-overrides ignore-differences` command is supposed to help users to test fields configured in `ignoreDifferences.jsonPointers` or `ignoreDifferences.jqPathExpressions`. The command accepts sample resource manifest, removes ignored fields and prints the difference to visualize which fields were ignored.

In order to test `managedFieldsManagers` user would have to provide sample git and sample live k8s resource, so we would need a separate command. But `managedFieldsManagers` is much easier to configure and there is no need so such command at all.

Reply:
Makes sense. Thanks for the explanation. Added proper comment in the code.